### DISCUSSION TITLE
PlotlyChart not show

### DISCUSSION THREAD

**Discussion by Lefort-Antoine** [elimintz/justpy#675] [Q&A]
Hello,

I have problem using PlotlyChart on serveur. My error is a Vus.js error 
"SyntaxError: JSON.parse: unexpected character at line 1 column 2 of the JSON data"

I suppose it is line 18 in templates/js/vue/plotlyjs.js : 
`const chart_obj = JSON.parse(this.$props.jp_props.chart);`

It work running the app on local. 
Do you have an idea?

here is the simple app:

`
import justy as jp

def demo_figure():
    wp = jp.QuasarPage()
    fig = go.Figure(
        data=[go.Bar(x=[1, 2, 3], y=[1, 3, 2])],
        layout=go.Layout(
            title=go.layout.Title(text="A Figure Specified By A Graph Object")
        )
    )
    jp.PlotlyChart(
        a=wp.container,
        style="max-width: 100%",
        chart=fig,
        )
    return wp
`

**Comment by WolfgangFahl** [comment]
looks like the json transmitted is empty.

**Reply by Lefort-Antoine** [reply]
You right on the online server : 
> this.$props.jp_props.chart.toString()
"[object Object]" 

Which correspond to empty data compare to the local one
I investigate because I exact same code.

**Reply by Lefort-Antoine** [reply]
The figure are show know. I did nothing...
